src/Pipl.php

```diff
@@ -13,42 +13,34 @@
  */
 class Pipl
 {
-
-    /**
-     * @var ClientInterface
-     */
     protected $client;
 
     public function __construct(ClientInterface $client = null)
     {
         $this->client = $client ?? (new Client());
     }
 
-    /**
-     * @param array $arrayOfFields
-     * @return mixed
-     * @throws Exception
-     */
-    public function search(array $arrayOfFields)
```

Comment:
Why did you removed the code block here ?

Reply:
What does that block do?  It is cluttering the code if you have a comment block commenting nothing you don't already know.

Reply:
It's a good practice to put docBlock above every function, because it's easier to see what the function does.It's also used to generate code documentation.
For more info please read https://docs.phpdoc.org/guides/docblocks.html

Reply:
I tend to think (and also every other refactoring book) that software documentation in that form is an anti pattern. If your function requires a comment to describe it, it is either doing too much or is named wrong. You can also check out [this link](https://dzone.com/articles/meaningless-docblocks). 
It's your repository though, do as you think is good :)

Reply:
Good point